### PR TITLE
Implement `isSelected` db prop - this can be applied to either newspapers or devices. Fixes #19.

### LINKED ISSUE #19: Load NYT on load

### PR DESCRIPTION
**Pull request by dannya**

### CODE REVIEW

File: db.js [hunk 1]
```diff
@@ -12,6 +12,7 @@ module.exports = {
     {
       id: 'NYT',
       name: 'New York Times',
+      isSelected: true,
```
Comment (pathikrit) [inline db.js:19]: Should we add a check that there is exactly 1 `isSelected`?

File: views/index.ejs [hunk 2]
```diff
@@ -79,5 +66,32 @@
...
+    }
+    $('#url').html(`<a href="${url}" target="_blank">${window.location.protocol + '//' + window.location.host + url}</a>`)
+    $('#emulator').prop('src', url)
+  }
+
+  // initialize emulation on load
+  const db = <%-JSON.stringify(db)%>;
+
+  emulate(
+    // if there is an `isSelected` newspaper, initialize in newspapers mode, otherwise initialize in devices mode
```
Comment (pathikrit) [inline views/index.ejs:90]: Perhaps move this comment to `db.js`?

**Review by pathikrit — APPROVED** [review]

File: db.js (hunk 1 above)
Comment (dannya) [reply db.js:19]: IMO a check isn't necessary - maybe a comment in `db.js` that makes it clear that only one `isSelected` will be respected?

File: views/index.ejs [hunk 3]
```diff
@@ -79,5 +66,32 @@
...
+    (db.newspapers.filter(item => item.isSelected).length > 0) ?
```
Comment (pathikrit) [inline views/index.ejs]:
```suggestion
    db.newspapers.some(item => item.isSelected) ? 'newspapers' : 'devices'
```

File: db.js (hunk 1 above)
Comment (dannya) [reply db.js:19]: Comment in `db.js` added.

File: views/index.ejs (hunk 2 above)
Comment (dannya) [reply views/index.ejs:90]: Kept this comment here, and also added a further comment in `db.js`.

**Review by pathikrit — APPROVED** [review]

**Pull request merged**